### PR TITLE
Feature/login azure ad

### PR DESCRIPTION
**Pull request by tvdw2000**

### CODE REVIEW

File: src/main/java/com/seatapp/controllers/LoginController.java [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    /**
+     * Check if the user can reach this api call.
+     *
+     * @param token the JWT token of the user that reached the url
+     * @return a http response OK
+     */
+    @GetMapping("healthcheck")
+    public ResponseEntity<Void> healthCheck(
+            final UsernamePasswordAuthenticationToken token) {
+        if (LOGGER.isDebugEnabled()) {
```
Comment (JariMoons) [inline src/main/java/com/seatapp/controllers/LoginController.java:55]: Are these loggers usefull? At the moment they don't log anything.

File: src/main/java/com/seatapp/services/LoggerService.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
+package com.seatapp.services;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class LoggerService {
```
Comment (JariMoons) [inline src/main/java/com/seatapp/services/LoggerService.java:6]: Debug in logger is not enabled at the moment, either enable debug logging or remove te loggers.

**Review by JariMoons — CHANGES_REQUESTED** [review]

File: src/main/java/com/seatapp/controllers/LoginController.java (hunk 1 above)
Comment (tvdw2000) [reply src/main/java/com/seatapp/controllers/LoginController.java:55]: They can be used to find bugs and see if the flow of the application is correct.

File: src/main/java/com/seatapp/services/LoggerService.java (hunk 2 above)
Comment (tvdw2000) [reply src/main/java/com/seatapp/services/LoggerService.java:6]: The loggers are now enabled.

**Review by JariMoons — APPROVED** [review]

**Pull request merged**